grr/server/grr_response_server/gui/root/api_root_router.py

```diff
@@ -101,3 +106,17 @@ def RestartFleetspeakGrrService(self, args, token=None):
   @api_call_router.NoAuditLogRequired()
   def ListApiMethods(self, args, token=None):
     return api_reflection.ApiListApiMethodsHandler(self)
+
+  # Metadata methiods.
+  # ===========================================================
+  #
+  @api_call_router.Category("Metadata")
+  @api_call_router.ResultType(api_metadata.ApiGetOpenApiDescriptionResult)
+  @api_call_router.Http("GET", "/api/metadata/openapi")
+  @api_call_router.NoAuditLogRequired()
+  def GetOpenApiDescription(
+      self,
+      args: None,
+      token: Optional[access_control.ACLToken] = None,
+  ) -> api_metadata.ApiGetOpenApiDescriptionHandler:
+    return api_metadata.ApiGetOpenApiDescriptionHandler()
```

Comment:
Oh, now I think I understand why you asked about this. I believe you don't need to implement this.

Reply:
I removed the implementation from `ApiRootRouter`. Thank you!